### DISCUSSION TITLE
about future development?

### DISCUSSION THREAD

**Discussion by paklie** [elementary/code#1085] [General]
for future development...is this app going to be a versatile code editor with support for much language like atom or vscode....or this app will focusing on vala only???

**Comment by VighneshNilajakar** [comment]
Ya, I think we can just add install IDE for Languages like Python and Integrate them with CODE app to work as this will become the go to place to do any sort of Programming I think this will be really Cool the app looks really cool thankyou developers for building such a Cool app.

**Comment by jeremypw** [comment]
I think the priority is to make an excellent tool for developing elementary apps (Vala/Meson/Git) rather than a mediocre tool for a variety of languages.  However, I would hope that the plugins and other architecture would allow for extending its use to other languages in future.  Progress will, of course, depend on rate of contribution of code from devs.
